data/human/Republic Paradise Threatened.TXT

```diff
@@ -0,0 +1,301 @@
+# Paradise Threatened
+
+mission "RN Analyze the Nukes"
+	Name "Analyze the Nukes"
+	description "Go to <destination> to drop the sensor readings off at Lovelace Labs to be analyzed."
+	deadline
+	source "Geminus"
+	destination "Ada"
+	on offer
+		conversation
+			`Amelia is engrossed in some kind of spreadsheet when you barge into her office and drop a datachip of scans from your fight with the Wolf Pack onto her desk. Her brows knit at this breach of protocol but she switches her display to show the data and you watch closely as her eyes widen and the colour drains from her face.`
+			`	“Is this… this can’t be right!" she says, flipping back and forth through the images. “Nukes? I... I almost believed them!"`

```

Comment:
I think what I was going for here was to really drive home to the player just how big a deal this is. You’re right that Shah is generally unflappable but the use of nukes is so far outside of her expectations that it shakes her.

Of course it may be that this isn’t necessary because it will be the second time that nukes are used, after the Gemini’s (and the other planet I can’t remember right now) bombings.

Reply:
Hahahaha, it just shows how much the bombings really don't have an impact, which is something I'm trying to help solve. The other planet is Martini.